students/WooseokJ/TrigramWJ.py

```diff
@@ -0,0 +1,37 @@
+import sys 
+import random
+
+#takes the words from the given file and returns it in a list
+def words_from_file(filename):
+    with open(filename, 'r') as textfile: 
+        return [word.strip('\n') for l in textfile.readlines() 
+                for word in l.split(' ')] 
+#from the list makes a dictiondary to mix it in the next process
+def build_trigrams_dict(words):
+    trigrams = {}
+    for i in range(len(words)-2):
+        pair = tuple(words[i:i + 2])
+        follower = words[i + 2]
+        if pair not in trigrams:
+            trigrams[pair] = [follower]
+        else:
+            trigrams[pair] += [follower]
+    return trigrams
+#this mixes the words to create the new text
+def build_text(pairs):
+    random_text = []
+    while len(random_text) < len(pairs):
+        random_key = random.choice(list(pairs.keys()))
+        value = pairs.get(random_key)
```

Comment:
An easier way to get the list of words associated to _random_key_ is to do _pairs[random_key]_.